bigtable-grpc-interface/pom.xml

```diff
@@ -43,35 +43,8 @@ limitations under the License.
             <artifactId>grpc-all</artifactId>
             <version>${grpc.version}</version>
         </dependency>
-        <dependency>
-            <groupId>io.netty</groupId>
-            <artifactId>netty-all</artifactId>
-            <version>${netty.version}</version>
-        </dependency>
```

Comment:
mvn dependency:analyze was flaging this netty-all dependency as a not used, and grpc's more specific use of netty-codec (and something else) were used.  I'll add those in.